<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyLockMessage</class>
 <widget class="QDialog" name="VerifyLockMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">MEGA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VerifyLockMessage
{
background-color: #ffffff; /*colorToken.page-background*/
}</string>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="IconLabel" name="bWarning">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources_common.qrc">
         <normaloff>:/alert-triangle_medium_thin_outline.svg</normaloff>:/alert-triangle_medium_thin_outline.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="normal_off" stdset="0">
        <string notr="true">support-warning</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="wContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lVerifyEmailTitle">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">title</string>
          </property>
          <property name="font-size" stdset="0">
           <string notr="true">label-2</string>
          </property>
          <property name="semibold" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lVerifyEmailDesc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="font-size" stdset="0">
           <string notr="true">body-2</string>
          </property>
          <property name="regular" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="TokenizableButton" name="bWhySeenThis">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Why am I seeing this?</string>
         </property>
         <property name="icon">
          <iconset resource="../Resources_common.qrc">
           <normaloff>:/external_link_small_thin_outline</normaloff>:/external_link_small_thin_outline</iconset>
         </property>
         <property name="type" stdset="0">
          <string notr="true">link</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bLogout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Log out</string>
         </property>
         <property name="type" stdset="0">
          <string notr="true">outline</string>
         </property>
         <property name="dimension" stdset="0">
          <string notr="true">small</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TokenizableButton</class>
   <extends>QPushButton</extends>
   <header>tokenizer/TokenizableItems/TokenizableButtons.h</header>
  </customwidget>
  <customwidget>
   <class>IconLabel</class>
   <extends>QToolButton</extends>
   <header>tokenizer/TokenizableItems/IconLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources_common.qrc"/>
 </resources>
 <connections/>
</ui>
